Common/ISubTaskFactory.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System;
```

Comment:
The name of the file is incorrect :slightly_frowning_face: It should be `ISubtaskFactory`